src/app/(drops)/projects/[project]/drops/new/details/page.tsx

```diff
@@ -19,7 +19,7 @@ const BLOCKCHAIN_LABELS = {
   [Blockchain.Polygon]: 'Polygon',
 };
 
-const BLOCKCHAIN_OPTIONS = [Blockchain.Solana];
+const BLOCKCHAIN_OPTIONS = [Blockchain.Solana, Blockchain.Polygon];
```

Comment:
We need some extra validation logic for polygon. The differences are:

Can only have 1 creator.
Can remove the max length constraints on name and symbol so switch the maxLength validation with a custom one.

And if you return a string it should show the error banner to resolve this bug ticket at the same time.

https://github.com/holaplex/hub/issues/215